test/metabase/dashboard_subscription_test.clj

```diff
@@ -175,21 +176,28 @@
                                          :description   "Linked Dashboard desc"
                                          :collection_id coll-id}
        DashboardCard _                  {:dashboard_id           dashboard-id
+                                         :row                    0
                                          :visualization_settings (link-card-viz-setting "collection" coll-id)}
        DashboardCard _                  {:dashboard_id           dashboard-id
+                                         :row                    1
                                          :visualization_settings (link-card-viz-setting "database" db-id)}
        DashboardCard _                  {:dashboard_id           dashboard-id
+                                         :row                    2
                                          :visualization_settings (link-card-viz-setting "table" table-id)}
        DashboardCard _                  {:dashboard_id           dashboard-id
+                                         :row                    3
                                          :visualization_settings (link-card-viz-setting "dashboard" dash-id)}
        DashboardCard _                  {:dashboard_id           dashboard-id
+                                         :row                    4
                                          :visualization_settings (link-card-viz-setting "card" card-id)}
        DashboardCard _                  {:dashboard_id           dashboard-id
+                                         :row                    5
                                          :visualization_settings (link-card-viz-setting "dataset" model-id)}
        DashboardCard _                  {:dashboard_id           dashboard-id
+                                         :row                    6
                                          :visualization_settings {:virtual_card {:display "link"}
                                                                   :link         {:url "https://metabase.com"}}}]
-      (thunk {:collection-owner-id rasta-pc-id
+      (thunk {:collection-owner-id rasta-id
```

Comment:
This is the caused of ` java.lang.AssertionError: Assert failed: Makes sure you wrapped this with a `with-current-user`.
Example failure: 
https://github.com/metabase/metabase/actions/runs/4315201911/jobs/7529265310#step:4:1249

previously this returned the `collection-id` but it means to return the `owner-id` instead.
This was accidentally passed because when creating test users the personal collection id usually has the same index as the user id.

Reply:
Oops!